cypress/integration/catalog/collections.js

```diff
@@ -216,7 +219,127 @@ filterTests({ definedTags: ["all"] }, () => {
       });
     });
 
-    it("should update collection", () => {
+    it("As an admin I should be able to delete several collections on collections list page. SALEOR_0309", () => {
```

Comment:
```suggestion
    it("to delete several collections on collections list page. SALEOR_0309", () => {
```
or smg different, but without this part "As an admin.."

Reply:
corrected